test/box-luatest/gh_7904_export_box_schema_version_to_public_api_and_lua_test.lua

```diff
@@ -0,0 +1,26 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new{
+        alias   = 'dflt',
+    }
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+--[[
```

Comment:
Nit: We don't use multi-line comments in Lua.

Reply:
Sorry, I copy-paste the test skelet from other tests and forget about this — fixed.